### PR TITLE
Fixes the ERT med pouch

### PR DESCRIPTION
**Pull request by Foxtrot13224**
# About the pull request

Fixes the ERT medical pouches being unable to hold items a normal medical pouch should be able to hold

# Explain why it's good for the game

The ERT medical pouch spawns in with injectors in it, being able to take them out but not place them back inside makes no sense, so this fixes that

# Changelog

:cl: foxtrot1322
fix: Fixed the issue with ERT first-aid pouches being unable to hold injectors once taken out
/:cl:

### CODE REVIEW

File: code/game/objects/items/storage/pouch.dm [hunk 1]
```diff
@@ -244,13 +244,11 @@
 	icon_state = "firstaid"
 	storage_slots = 5
 	can_hold = list(
```
[the remark this answers is not in the document]
Comment (fira) [reply code/game/objects/items/storage/pouch.dm:246]: indeed

**Review by fira — CHANGES_REQUESTED** [review]
Please remove the can_hold entirely as indicated in comments as it is redundant

File: code/game/objects/items/storage/pouch.dm (hunk 1 above)
[the remark this answers is not in the document]
Comment (Foxtrot13224) [reply code/game/objects/items/storage/pouch.dm:246]: Wilco

**Review by fira — APPROVED** [review]
neato 👍

**Pull request merged**